src/components/ResourceList/ResourceList.tsx

```diff
@@ -323,20 +323,22 @@ export class ResourceList extends React.Component<CombinedProps, State> {
   }
 
   componentDidMount() {
-    this.setState(
-      {
-        listNode: this.listRef.current,
-      },
-      () => {
-        if (this.props.loading) {
-          this.setLoadingPosition();
-        }
-      },
-    );
+    this.forceUpdate();
```

Comment:
I agree that holding the node in state may not be the best approach. Do you prefer using forceUpdate like this or maybe a boolean of `hasMounted` or `refsMounted`, something a little more explicit of its purpose?